app/models/FileRole.scala

```diff
@@ -49,7 +49,7 @@ object FileRole {
         case "duty-deferment" => Right(DutyDefermentStatement)
         case "adjustments" => Right(SecurityStatement)
         case "postponed-vat" => Right(PostponedVATStatement)
-        case "cash-statement" => Right(CashStatement)
+        case "cash-statement" => Right(CDSCashAccount)
```

Comment:
Could you please add test case for this and change at line 63 as well? If it takes time then leave it

Reply:
Added tests